<?xml version="1.0" encoding="UTF-8" ?>

<launch>
     <include file="$(find vrep_ros_teleop)/launch/teleop_mux.launch"/>

    <node pkg="shore_follower" type="shore_follower_drive.py" name="shore_follower_drive" output="screen">
      <param name="model_dir" value="$(find shore_follower)/models/models/best_model" />
      <!-- rotation speed corresponding to left or right commands -->
      <param name="twist_factor" value="0.5" />
      <param name="linear_vel" value="0.2" />

      <remap from="/shore_follower_drive/image" to="/vrep/visionSensor"/>
      <remap from="/shore_follower_drive/twist" to="/mux/autoCommand"/>
    </node>
    
</launch>
